### DISCUSSION TITLE
Custom data images

### DISCUSSION THREAD

**Discussion by albertofernandezvillan** [AlexeyAB/darknet#8333] [Q&A]
I am creating some images to train a custom object detector with YOLO V4. 
For example, suppose I want to train an object detector for detecting bicycles in thermal images, and also, I want to create the minimal number of images for training/testing/validating. For this purpose, I am performing "artificial images" with objects over a real background. See the two attached images as an example. 
The question is: Is it a good idea to have this kind of images or is it a bad idea?
Many thanks in advance

![Bicis_03](https://user-images.githubusercontent.com/6097318/149941699-286a236c-dd60-4def-b80d-58db3159e4b9.jpg)
![Bicis_05](https://user-images.githubusercontent.com/6097318/149941738-dab479e8-dd3b-4903-aeec-281cb71a5c0d.jpg)

**Comment by stephanecharette** [comment]
Note that darknet already has built-in support for mosaic.  You simply have to turn it on in the .cfg file.  For example:  https://www.ccoderun.ca/darkmark/DataAugmentationMisc.html#Mosaic

**Reply by albertofernandezvillan** [reply]
Thank you for the answer. I am aware of this feature. This is why I have asked it. If I am not wrong mosaic data augmentation combines 4 images into one and different scales and ratios allowing the model to learn how to identify objects at different scales.
Therefore, I don't know if creating the images for training that I have introduced in this question is a good idea or not. I suppose it is not bad, but I am looking for feedback from other users.
